chapters/ch03.md

```diff
@@ -8,7 +8,7 @@ Nếu bạn đã từng học một lớp về Machine Learning ví dụ như l
 
 > I assume you are familiar with **supervised learning**: learning a function that maps from x to y, using labeled training examples (x,y). Supervised learning algorithms include linear regression, logistic regression, and neural networks. There are many forms of machine learning, but the majority of Machine Learning’s practical value today comes from supervised learning.
 
-Tôi giả định rằng bạn đã quen thuộc với **học có giám sát**: học một phương trình ánh xạ từ x tới y, sử dụng các cặp dữ liệu có nhãn (x,y). Các thuật toán học có giám sát bao gồm hồi quy tuyến tính, hồi quy logistic và mạng neural. Machine Learning có rất nhiều nhóm tuy nhiên phần lớn các giá trị thực tiễn của nó hiện nay đến từ học có giám sát.
+Tôi giả định rằng bạn đã quen thuộc với học có giám sát: học một hàm ánh xạ từ x tới y, sử dụng các cặp dữ liệu có nhãn (x,y). Các thuật toán học có giám sát bao gồm hồi quy tuyến tính, hồi quy logistic và mạng neural. Machine Learning có rất nhiều dạng tuy nhiên phần lớn các giá trị thực tiễn của nó hiện nay đến từ học có giám sát.
```

Comment:
cụm **supervised learning** trong bản gốc được in đậm, bản dịch lần đầu cũng được in đậm. Bạn có thể cho biết lý do bạn bỏ in đậm không?

Reply:
Ah cái này mình nhầm, không thấy ký hiệu in đậm

Reply:
> Ah cái này mình nhầm, không thấy ký hiệu in đậm

Chính vì vậy mình mới ưu tiên mọi thay đổi phải được tạo trong một PR riêng và review kỹ, không thì sẽ lỡ tay xoá hết công sức review trong các PR cũ.